ankihub/gui/overview.py

```diff
@@ -53,15 +56,10 @@ def _maybe_add_flashcard_selector_button() -> None:
     if not aqt.mw.state == "overview":
         return
 
-    # Only add the button if the currently open deck overview is for the Anking deck or a child of it
-    anking_deck_config = config.deck_config(config.anking_deck_id)
+    ah_did = get_ah_did_of_deck_or_ancestor_deck(aqt.mw.col.decks.current()["id"])
```

Comment:
We need to consider the ancestor decks as well, because the flashcard selector should also appear when you click on a subdeck of an AnkiHub deck (which as note embeddings).

Reply:
Thank you. I didn't remember about the ancestor decks.